src/components/grid/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { SimpleGrid, SimpleGridProps } from '@chakra-ui/react';
+
+type Props = Pick<
+  SimpleGridProps,
+  'minChildWidth' | 'columns' | 'spacing' | 'spacingX' | 'spacingY'
+>;
+
+const Grid: FC<Props> = (props) => <SimpleGrid {...props} />;
```

Comment:
Is it make sense to follow what we have done in the previous components regarding the chakra component name?

```suggestion
const Grid: FC<Props> = (props) => <ChakraSimpleGrid {...props} />;
```

Reply:
We have it because otherwise, the names would've clashed. Do you think it's still valuable to add it without the name clash?

Reply:
I see your point, so then let us keep it as you have. Thanks